Libs/Image/Image.cpp

```diff
@@ -286,10 +286,29 @@ bool Image::resample(float isoSpacing, PixelType defaultvalue, Dims outputSize)
 ///////////////////////////////////////////////////////////////////////////////
 bool Image::compare_equal(const Image &other)
 {
+  // we use the region of interest filter here with the full region because our
```

Comment:
Is there another comparison function? How do we now test functions like `recenter` which do not modify the image, only its origin?

Reply:
I believe it will still work for a function such as `recenter`.